var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -29,6 +29,7 @@
 from spack.pkg.builtin.intel import IntelInstaller, filter_pick, \
     get_all_components
 
+from llnl.util.filesystem import mkdirp
```

Comment:
You shouldn't actually need to import this. It should already be imported by `from spack import *`.